account_payment_mode/models/account_payment_mode.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import ValidationError
+
+
+class AccountPaymentMode(models.Model):
+    """This corresponds to the object payment.mode of v8 with some
+    important changes. It also replaces the object payment.method
+    of the module sale_payment_method of OCA/e-commerce"""
+    _name = "account.payment.mode"
+    _description = 'Payment Modes'
+    _order = 'name'
+
+    name = fields.Char(string='Name', required=True)
```

Comment:
This one should be also translatable for having different names depending on UI language.